plugin/pkg/global/customverbauthorizer/admission.go

```diff
@@ -31,6 +31,13 @@ const (
 	// CustomVerbProjectManageMembers is a constant for the custom verb that allows to manage human users or
 	// groups subjects in the `.spec.members` field in `Project` resources.
 	CustomVerbProjectManageMembers = "manage-members"
+
+	// CustomVerbNscpflModifyKubernetes is a constant for the custom verb that allows modifying the
+	// `.spec.kubernetes` field in `NamespacedCloudProfile` resources.
+	CustomVerbNscpflModifyKubernetes = "modify-spec-kubernetes"
+	// CustomVerbNscpflModifyMachineImages is a constant for the custom verb that allows modifying the
+	// `.spec.machineImages` field in `NamespacedCloudProfile` resources.
+	CustomVerbNscpflModifyMachineImages = "modify-spec-machineimages"
```

Comment:
Do not use uncommon abbreviations like` Nscpfl`. Either remove this part, or call it `NamespacedCloudProfile`.